CONTRIBUTING.md

```diff
@@ -3,15 +3,16 @@
 1.  Request access to this repo
 
 1.  Pull the latest master
-    - On the `master` branch: `git pull`
+
+    * On the `master` branch: `git pull`
```

Comment:
@jasmeet12 This file should not be in this PR. To revert it back to what is on the master branch, use this command:
`git checkout origin/master CONTRIBUTING.md`

You'll want to make note of that command, as it's very useful to revert files back to what is on the master branch.